inclusive-design/wecount.inclusivedesign.ca
# feat: switch to Eleventy

## Description

This PR implements the Eleventy static site generator. This is a breaking change.

## Steps to test

1. Visit deploy preview.

**Expected behavior:** Site matches existing production site.

## Additional information

- https://www.11ty.dev/docs/

## Related issues

Not applicable.
